src/fastener/item.py

```diff
@@ -9,7 +9,7 @@
 import sklearn.feature_selection
 import numpy as np
 
-from src import random_utils
+import fastener.random_utils as random_utils
```

Comment:
Apart from keeping dummy data in a separate folder, is there a reason to move fastener into `src` directory? 

It results in the example file only working with the library installed, but not with the library from the cloned repository.